.vscode/settings.json

```diff
@@ -1,3 +1,5 @@
 {
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "files.associations": {
+    "*.mdx": "markdown"
```

Comment:
mdx is not technically markdown

have you tried the MDX plugin?

<img width="347" alt="image" src="https://github.com/user-attachments/assets/15d3c32e-3a9f-4e64-9b83-6488fda61ee5">

Reply:
Thank you, plugin fixed it.